src/main/java/seedu/duke/CommandManager.java

```diff
@@ -111,19 +111,20 @@ public void commandChecker() throws DukeException, NullPointerException, MealExc
             }
             break;
         case Keywords.INPUT_DELETE_WEIGHT:
-            try {
-                weightTracker.readInput(input);
-            } catch (DukeException e) {
-                return;
-            }
-            break;
+            //removed the below code as they have identical functions
+            //try {
+            //    weightTracker.readInput(input);
+            //} catch (DukeException e) {
+            //return;
+            //}
+            //break;
```

Comment:
Looks good, case statements without anything will just assume the functionality of the next nearest non empty case block